scroll-animations-1/Overview.bs

```diff
@@ -353,18 +354,105 @@ spec: cssom-view-1; type: dfn;
 
 	Values are as defined for ''scroll()''.
 
+	The [=used value=] of 'scroll-timeline-axis'
+	is the axis of the [=current attachment=],
+	or the initial value if there is no [=current attachment=].
+
+### Attachment of a Scroll Progress Timeline: the 'scroll-timeline-attachment' property ### {#scroll-timeline-attachment}
+
+	<pre class='propdef'>
+	Name: scroll-timeline-attachment
+	Value: [ local | defer | ancestor ]#
+	Initial: local
+	Applies to: all elements
+	Inherited: no
+	Computed value: a list of the keywords specified
+	Animation type: not animatable
+	</pre>
+
+	Specifies attachment behavior for each [=scroll progress timeline=]
+	associated with this element.
+
+	A scroll timeline has a set of <dfn>attachments</dfn>,
+	that determine which [=scroll container=] and {{ScrollTimeline/axis}} to use
+	when calculating the [progress of the timeline](#scroll-timeline-progress).
+
+	<dl>
+		<dt><dfn>'local'</dfn>
+		<dd>
+			The [=attachments=] of the timeline is a single item set
+			containing that timeline.
+
+		<dt><dfn>'defer'</dfn>
+		<dd>
+			The [=attachments=] is the set of timelines in the flat-tree descendants
+			that satisfy all of the following:
+			 - The timeline has a matching 'scroll-timeline-name'.
+			 - The timeline has [=ancestor=] attachment.
+			 - The timeline has no intermediate [=defer|deferred=] timeline
+			   that would capture it.
+
+		<dt><dfn>'ancestor'</dfn>
+		<dd>
+			The set of [=attachments=] is empty.
+
+			Timelines with [=ancestor=] attachment are treated as nameless
+			for the purposes of timeline lookup;
+			they can't be referenced by 'animation-timeline'.
+	</dl>
+
+	The <dfn>current attachment</dfn> of a timeline is determined as follows:
+
+	<dl class="switch">
+		:   If the set of [=attachments=] contains exactly one timeline
+		::  The [=current attachment=] is that timeline.
+
+		:   Otherwise,
+		::  There is no [=current attachment=].
+	</dl>
+
+	<div class=example>
+		Using a [=defer|deferred timeline=],
+		an element can effectively refer to timelines
+		that appear <em>after it</em> in tree order.
+		For example, the following creates an animation on <code>#target</code> that
+		is ultimately linked to a scroll timeline defined on the subsequent sibling.
+
+		<pre class=lang-css>
+		@keyframes anim {
+		  from { color: red; }
+		  to { color: green; }
+		}
+
+		#root {
+		  scroll-timeline-name: mytimeline;
+		  scroll-timeline-attachment: defer;
+		}
+
+		#root #target {
+			animation: anim auto;
+			animation-timeline: mytimeline;
+		}
+
+		#root #target + #scroller {
+			scroll-timeline-name: mytimeline;
+		  scroll-timeline-attachment: ancestor;
```

Comment:
nit: inconsistent indentation in this code sample.

Reply:
Thanks, done.